requirements.txt

```diff
@@ -156,3 +156,4 @@ whitenoise==6.3.0
 
 # The following packages are considered to be unsafe in a requirements file:
 # setuptools
+setuptools>=65.5.1 # not directly required, pinned by Snyk to avoid a vulnerability
```

Comment:
If we want to persist changes to the requirements files, they need to be made in the .in files and then `pip-compile`d, else they'll be overwritten the next time someone runs a pip-compile. Individual packages (like `cryptography` in this case) can be upgraded with the pip-compile `--upgrade` command: https://github.com/jazzband/pip-tools#updating-requirements

Re: the setuptools change, we could put a >= version req in the requirements.in file, but explicitly declaring setuptools in a requirements file is generally not a good idea, as it makes installing our dependencies very finicky with anything other than the exact version of python we might be using. Installing setuptools separately is the recommended approach to ensure that someone ends up with a version that's compatible (https://github.com/broadinstitute/seqr/blob/master/deploy/docker/seqr/Dockerfile#L39)

Given the note here from the description of the CVE, I'd say that it might be a better option to just tell snyk to ignore this.

> Only a small portion of the user base is impacted by this flaw. Setuptools maintainers pointed out that package_index is deprecated (not formally, but “in spirit”) and the vulnerability isn't reachable through standard, recommended workflows.